### ISSUE TITLE
[BUG] Linking fails when using the latest AppCenter package 2.6.2

### ISSUE THREAD

**Issue by winterdouglas** [RLittlesII/serilog-sinks-appcenter#1]
**Describe the bug**
When updating the AppCenter packages to 2.6.2, linking fails for both Android and iOS with the following messages:
iOS:
```
MTOUCH : error MT2101: Can't resolve the reference 'System.Void Microsoft.AppCenter.Crashes.Crashes::TrackError(System.Exception,System.Collections.Generic.IDictionary`2<System.String,System.String>)', referenced from the method 'System.Void Serilog.Sinks.AppCenter.Crash.CrashSink::TrackError(System.Exception,System.Collections.Generic.IDictionary`2<System.String,System.String>)' in 'Microsoft.AppCenter.Crashes, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
```

Android:
```
error MSB4018: Mono.Linker.MarkException: Error processing method: 'System.Void Serilog.Sinks.AppCenter.Crash.CrashSink::TrackError(System.Exception,System.Collections.Generic.IDictionary`2<System.String,System.String>)' in assembly: 'Serilog.Sinks.AppCenter.Crash.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve System.Void Microsoft.AppCenter.Crashes.Crashes::TrackError(System.Exception,System.Collections.Generic.IDictionary`2<System.String,System.String>)
```

- I looked at the code here in the repo and the package dependency for `AppCenter.Crashes` is 2.* which would mean that any version 2 would do, however, looking the min version on nuget it says that 2.5 is the minimum to be used. How's that even possible?
- If it states on nuget that 2.5 is the minimum why would it be failing with 2.6.2? that should be compatible. Apparently there's something weird going on.
- I cloned the repo here referenced `AppCenter.Crashes` 2.6.2 directly and there were no breaking changes or something that could affect, I was able to build the project right away.
- With the version that I built myself the same error doesn't happen.

**Steps To Reproduce**
- In a Xamarin.Forms app install the `Serilog.Sinks.AppCenter.Crash` 1.2.0
- Install `Microsoft.AppCenter.Crashes` 2.6.2
- Enable at least linking SDKs
- Build
- It will fail

**Comment by stepkillah** [comment]
Are there any plans to fix this one?  @RLittlesII

**Comment by RLittlesII** [comment]
@stepkillah Yes.  I actually just reached out to @winterdouglas about reproduction.  I am going to be looking into the problem this week.

**Comment by stepkillah** [comment]
@RLittlesII cool, thanks a lot

**Comment by RLittlesII** [comment]
@winterdouglas Can you verify the fix on [GH-1](https://github.com/RLittlesII/serilog-sinks-appcenter/tree/gh1)

I didn't really do much other than update to the latest 2.6.4 dependency.

**Comment by winterdouglas** [comment]
Hey @RLittlesII.
I'm able to build normally here on both Android and iOS in release with linking sdk enabled.

I cloned your branch, built a package and installed it in my app from a local feed as there's no updated package yet.

Your change solved the issue.

Thanks!

**Comment by RLittlesII** [comment]
Resolved: bb2feaad5e6d751d16c4f2bffb07450f1b496651
